lib/Data/Chronicle/Writer.pm

```diff
@@ -69,19 +69,31 @@ to save data and another method to retrieve it. All the underlying complexities
 
 =cut
 
+=head1 Members
```

Comment:
Same comments in this file too about

    =head1 METHODS

Also "instance of", use of `L<...>` to refer to other class names